src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -38,29 +48,20 @@ class ValidateActor(wdlSource: WdlSource, workflowInputs: Option[WdlJson], workf
   override def receive = {
     case ValidateWorkflow =>
       validateWorkflow(sender())
-      // NOTE: self shuts down when the parent PerRequest shuts down
-  }
-
-  private def validateInputs(namespaceWithWorkflow: NamespaceWithWorkflow, maybeWorkflowInputs: Option[WdlJson]): Future[Unit] = {
-    maybeWorkflowInputs match {
-      case Some(wi) =>
-        for {
-          inputs <- Future(wi.parseJson).map(_.asJsObject.fields)
-          coercedInputs <- Future.fromTry(namespaceWithWorkflow.coerceRawInputs(inputs))
-        } yield ()
-      case None => Future.successful(())
-    }
+    // NOTE: self shuts down when the parent PerRequest shuts down
   }
 
   private def validateWorkflow(sentBy: ActorRef): Unit = {
     logger.info(s"$tag for $sentBy")
-    val futureValidation: Future[Unit] = for {
+    val futureValidation: Future[ValidationSuccess] = for {
```

Comment:
This should be `Future[ValidationResult]`, with the result of the forcomp `recover`ed to produce a `ValidationFailure`:

```scala
val futureValidation: Future[ValidationResult] = (for {
 ... 
} yield ValidationSuccess(...)) recover { case e => ValidationFailure(e) } 
```